docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+# Configuration for the services that are required to run an analysis of the
+# data from riff edu mattermost.
+# see https://www.metabase.com/docs/latest/operations-guide/running-metabase-on-docker.html
+version: '3.6'
+services:
+
+  target-db:
+    image: mysql:5.7
+    container_name: mysql-business-analytics
```

Comment:
why is there a `container_name` instead of just letting docker-compose create the container name from the directory and the service name?

Reply:
I just thought it was nice to provide a nice container name, the generated one seemed somewhat cryptic. And I need to refer to the container name inside the script in /bin to restore the sql backups.

Reply:
Those are both an interesting and reasonable points.

However, I don't think we should care how cryptic the container name looks, as long as it is identifiable as to what it is.

As for referring to the container from inside your script, that is a reasonable initial approach. I think longer term we will run into issues doing it that way, but it does solve an otherwise tricky problem. Since you've followed my other naming suggestions for the services, I would suggest at least setting the container name to match that convention. (And add a comment as to why you're setting the container name)

Reply:
Removed container name from the metabase container since I don't reference it explicitly. Changed the mysql container name to match conventions and added a comment explaining where the name is used.